custom/icds_reports/static/js/directives/exclusive-breastfeeding/exlusive-breastfeeding.directive.js

```diff
@@ -2,11 +2,16 @@
 var url = hqImport('hqwebapp/js/initial_page_data').reverse;
 
 function ExclusiveBreasfeedingController($scope, $routeParams, $location, $filter, maternalChildService,
-    locationsService, dateHelperService, navigationService, userLocationId, storageService, genders, haveAccessToAllLocations, baseControllersService, isAlertActive) {
+    locationsService, dateHelperService, navigationService, userLocationId, storageService, genders, haveAccessToAllLocations,
+    baseControllersService, isAlertActive, isMobile) {
     baseControllersService.BaseController.call(this, $scope, $routeParams, $location, locationsService,
-        dateHelperService, navigationService, userLocationId, storageService, haveAccessToAllLocations);
+        dateHelperService, navigationService, userLocationId, storageService, haveAccessToAllLocations,
+        false, isMobile);
```

Comment:
this `false` is an example of the `haveAccessToFeatures` flag not actually having been passed to the base controller at all that I mentioned earlier. I'm just defaulting it to something falsey which it already was rather than pass it up the chain which would have added extra noise to this PR.